app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict[str]) -> str | Exception:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+        if visitor["vaccine"].get("expiration_date") < date.today():

```

Comment:
visitor["vaccine"].get("expiration_date") < date.today()
compose None and date is not possible